internal_transfers/actions/src/parse.ts

```diff
@@ -91,3 +97,27 @@ export function tryParseERC20Transfer(log: Log): TransferEvent | null {
     amount: BigInt(value),
   };
 }
+
+export function tryParseWethAction(log: Log): TransferEvent | null {
+  if (log.address !== WETH_TOKEN_ADDRESS) return null; // Not emitted by WETH Contract!
+  const eventLog = wethInterface.parseLog(log);
+  if (
+    eventLog === null ||
+    !["Deposit", "Withdrawal"].includes(eventLog.name) ||
+    eventLog.args.address
```

Comment:
Why are you checking that `args.address` is empty? It isn't part of a `Deposit` and `Withdrawal` event so it's expected to be `undefined`, but it looks a bit arbitrary.

Reply:
removed in latest commit. Thanks.